### ISSUE TITLE
Bring back FTDI support

### ISSUE THREAD

**Issue by Maxwe11** [NModbus4/NModbus4#76]
Investigate how we can bring back FTDI support

**Comment by niarbeht** [comment]
Just for reference, part of the answer is to yell at me.

**Comment by ghost** [comment]
Yes. USB support also of interest for iot-core devices... Vociferously :-)

**Comment by niarbeht** [comment]
I can probably see about uploading a copy of my old NModbus version where I have the cross-platform FTDI capability.  I haven't tested it in forever, but it shouldn't have stopped working.
